ad-click/ad-click-impl/src/main/java/com/duckduckgo/adclick/impl/AdClickAttributionFeature.kt

```diff
@@ -16,11 +16,13 @@
 
 package com.duckduckgo.adclick.impl
 
+import com.duckduckgo.feature.toggles.api.FeatureExceptions.FeatureException
+
 data class AdClickAttributionFeature(
     val state: String,
     val minSupportedVersion: Int?,
     val settings: AdClickAttributionSettings,
-    val exceptions: List<AdClickAttributionException>,
+    val exceptions: List<FeatureException>,
```

Comment:
Nit: to be clear, this is fine. 
It feels odd for now because all these features (ad-click) etc related to the `privacy-config` module but take the model from the `feature-toggles` module.
No need to change, just a comment